scripts/deploy-backend.sh

```diff
@@ -16,17 +16,6 @@ generate_jwt_cert()
     yes 'XX' | openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes -sha256
     cf set-env "$CGHOSTNAME_BACKEND" JWT_CERT "$(cat cert.pem)"
     cf set-env "$CGHOSTNAME_BACKEND" JWT_KEY "$(cat key.pem)"
-
-    # make sure that we have something set that you can later override with the
-    # proper value so that the app can start up
-    if cf e "$CGHOSTNAME_BACKEND" | grep -q OIDC_RP_CLIENT_ID ; then
```

Comment:
This is no longer needed since we can effectively set defaults for this per space in Cloud.gov using the new settings configurations.